tests/test_acl.py

```diff
@@ -184,6 +184,17 @@ def verify_acl_rule(self, dvs, field, value):
             else:
                 assert False
 
+    def create_redirect_action_acl_rule(self, table_name, rule_name, qualifiers, intf, priority="2020"):
+        fvs = {
+            "priority": priority,
+            "REDIRECT_ACTION": intf
+        }
+
+        for k, v in qualifiers.items():
+            fvs[k] = v
+
+        self.config_db.create_entry("ACL_RULE", "{}|{}".format(table_name, rule_name), fvs)
```

Comment:
Not available in 201911 branch

Reply:
@wendani, since in 201911 branch, vstests were not working then, I did not include my test case when it was merged to 201911. @abdosi knows about it.